assignments/callbacks.js

```diff
@@ -1,29 +1,54 @@
+
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
+const lazyMe = function(squrrel) {
+  console.log(squrrel);
+}
+
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  cb(arr[0]);
 }
+firstItem(items, lazyMe);
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
+getLength(items, lazyMe);
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[arr.length - 1]);
+}
+last(items, lazyMe);
+function makeAddition(cats) {
```

Comment:
you could have kept using `lazyMe` instead of creating another function that does the same thing as `lazyMe`

BE LAZIER 😪